elliott/tests/test_util.py

```diff
@@ -126,6 +126,21 @@ def test_get_golang_container_nvrs_builder(self):
         actual = util.get_golang_container_nvrs(nvrs, None)
         self.assertEqual(expected, actual)
 
+    def test_suffix_replacement(self):
```

Comment:
Not every test needs to be its own function. You can have a single `test_fix_summary_suffix` method, that has something like

```python
testdata = [
  ["Bug is fine [openshift-4.12]", "[openshift-4.12]",  "Bug is fine [openshift-4.12]"],
  [ ... ]
]
for summary, version, expected in testdata:
    result = ...
    self.assertEqual(result, expected)
```